test/computer_player_test.exs

```diff
@@ -40,6 +40,13 @@ defmodule ComputerPlayerTest do
       move = ComputerPlayer.select_move(context[:initial_state])
       assert Map.fetch!(ComputerPlayer.available_moves(context[:initial_state]), move)
     end
+
+    test "enters move into board state", context do
+      computer_move_state = ComputerPlayer.make_move(context[:initial_state])
+      board_values = Map.values(computer_move_state.board)
+
+      assert Enum.find(board_values, fn x -> x == "X" end) == "X"
```

Comment:
There's a lot going on in this test to assert that one space on the board is "X". I wonder if there might be a different way to structure the code to make it easier to test.